ssl/d1_srtp.c

```diff
@@ -207,11 +207,15 @@ static int ssl_ctx_make_profiles(const char *profiles_string,
 
 int SSL_CTX_set_tlsext_use_srtp(SSL_CTX *ctx, const char *profiles)
 {
+    if (ctx->srtp_profiles)
```

Comment:
The line below can end up failing, leaving a freed ctx->srtp_profiles pointer around => double free.

Best integrate this into the static method.

Also, we should document what happens if this fails: are the existing profiles kept, or are there now no profiles. (I think I prefer the latter.)